modules/cas_cache/layer_cache_management.c

```diff
@@ -854,6 +854,11 @@ int cache_mngt_purge_device(const char *cache_name, size_t name_len)
 	if (result)
 		return result;
 
+	if (ocf_cache_is_standby(cache)) {
```

Comment:
Don't we want to handle this in OCF? IMO this check doesn't belong to the adapter layer.